lib/libutils/isoc/bget_malloc.c

```diff
@@ -115,18 +115,12 @@
 
 static uint32_t malloc_lock(void)
 {
-	uint32_t exceptions;
-
-	exceptions = thread_mask_exceptions(
-			THREAD_EXCP_NATIVE_INTR | THREAD_EXCP_FOREIGN_INTR);
-	cpu_spin_lock(&__malloc_spinlock);
-	return exceptions;
+	return cpu_spin_lock_xsave(&__malloc_spinlock);
```

Comment:
Yep, I did the mistake last time I patched that (previous version was also by me). It is okay to mask async aborts.